src/action/ListAction.java

```diff
@@ -0,0 +1,48 @@
+package action;
+
+import java.util.List;
+
+import com.opensymphony.xwork2.ActionSupport;
+
+import beans.CorrectAnswersBean;
+import beans.QuestionsBean;
+import dao.CorrectAnswersDAO;
+import dao.QuestionsDAO;
+
+public class ListAction extends ActionSupport{
+
+	List<QuestionsBean> qlist;
+	List<CorrectAnswersBean> calist;
+
+	public void initializeQlist() {
+		qlist = QuestionsDAO.getAllQuestions();
```

Comment:
QuestionsDAOのインスタンス化は不要でしょうか…？
```suggestion
		QuestionsDAO dao = new QuestionsDAO();
		qlist = dao.getAllQuestions();
```
例えばこんな。

Reply:
staticでメソッドを作っているのでインスタンス化せず呼び出せるようになっています。
あんまりstaticで作らない方がいいでしょうか？

Reply:
staticならこういう書き方になりませんか…？
```suggestion
		qlist = QuestionsDAO::getAllQuestions();
```

Reply:
PHPと混在しちゃってた。それでOKでした。すんません。

Reply:
そういえば、phpでその辺意識してませんでした。
逆に勉強になりました。